src/test/java/com/google/api/codegen/testdata/go_example_library.baseline

```diff
@@ -20,7 +20,7 @@ package library_test
 import (
     "cloud.google.com/go/library/apiv1"
     "golang.org/x/net/context"
-    googleapis_example_library_v1 "google.golang.org/genproto/googleapis/example/library/v1"
+    pb "google.golang.org/genproto/googleapis/example/library/v1"
```

Comment:
The `pb` name is a suggestion from @okdave. I think the reasoning is along the line of "we are writing client for pubsub, so pb is just protobuf of pubsub". I think it's the context of the code that makes it work.